### PR TITLE
perf(merge): remove unnecessary wrapping of jqLite element

### PR DESCRIPTION
**Pull request by gkalpak**
Fixes https://github.com/angular/angular.js/commit/75292a6cb5e17d618902f7996e80eb3118eff7b0#commitcomment-14137538

### CODE REVIEW

File: protractor-conf.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ config.specs = [
 ];
 
 config.capabilities = {
-  browserName: 'chrome'
+  browserName: 'firefox'
```
Comment (petebacondarwin) [inline protractor-conf.js]: Why this change?

**Comment by petebacondarwin** [comment]
One comment above. Otherwise LGTM please merge

File: protractor-conf.js (hunk 1 above)
Comment (gkalpak) [inline protractor-conf.js]: Ooops. I committed it by accident (I am using it locally, because of the problem I have with protractor+latest Chrome).

**Comment by gkalpak** [comment]
Backported to `v1.4.x` as ce6a96b.

**Comment by petebacondarwin** [comment]
Smashing, thanks
